### PR TITLE
Start setting up Viper converter

### PR DESCRIPTION
**Pull request by jesyspa**

### CODE REVIEW

File: compiler/frontend.common/src/org/jetbrains/kotlin/diagnostics/KtDiagnosticFactoryDsl.kt
Comment (gabryon99) [inline compiler/frontend.common/src/org/jetbrains/kotlin/diagnostics/KtDiagnosticFactoryDsl.kt:1]: Was this generated by the compiler?

File: gradle/verification-metadata.xml [hunk 1]
```diff
@@ -10136,6 +10136,12 @@
...
          </artifact>
       </component>
+      <component group="org.scala-lang" name="scala-reflect" version="2.13.0">
+         <artifact name="scala-reflect-2.13.0.jar">
+            <md5 value="cbbeec15c3444405743aba51dfe4fd0d" origin="Generated by Gradle"/>
+            <sha256 value="aef72378abbc8b05cfdc7144fc0e55904dd87bf46ce2e973ae667dc3ed2fcee8" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="org.scala-lang" name="scala-reflect" version="2.13.10">
          <artifact name="scala-reflect-2.13.10.jar">
```
Comment (gabryon99) [inline gradle/verification-metadata.xml:10146]: I think this could be a possible `scala-reflect` dependency conflict

Comment (jesyspa) [reply gradle/verification-metadata.xml:10146]: It doesn't seem to cause any issues. From what I understand, it allows the artefact to have either of the generated checksums.

Comment (gabryon99) [reply gradle/verification-metadata.xml:10146]: Perfect, then it makes sense! :)

**Pull request merged**